analysis/filter/MergeFilter/src/main/kotlin/de/maibornwolff/codecharta/filter/mergefilter/MergeFilter.kt

```diff
@@ -24,7 +24,7 @@ class MergeFilter(
     @CommandLine.Option(names = ["-h", "--help"], usageHelp = true, description = ["displays this help and exits"])
     var help: Boolean = false
 
-    @CommandLine.Parameters(arity = "1..*", paramLabel = "FILE or FOLDER", description = ["files to merge"])
+    @CommandLine.Parameters(arity = "0..*", paramLabel = "FILES", description = ["cc.json files to merge"])
```

Comment:
Why 0 or more and why just FILE and no longer FOLDER? Shouldn't you always have to pass in at least two files or one folder? What is there to merge if you pass in no files or folders?

Reply:
You are absolutely right. At least two files or one folder should always be required for a valid merge operation. The change allowing 0 or more files and accepting only FILE was temporarily made for testing purposes. It will be ensured that at least two files or one folder are provided to guarantee valid input.